components/progress-indicator/index.jsx

```diff
@@ -156,14 +156,23 @@ function checkSteps(steps) {
 }
 
 /**
- * Check if an item is from an array of items when 'items' is an array;
- * Check if an item is equal to the other item after being stringified when 'items' is a JSON object
+ * Check if an item is from an array of items.
+ * If items argument is not an array, it will be treated as an object comparison between item & items.
  */
 function findStep(item, items) {
```

Comment:
@interactivellama I tweaked this to avoid JSON.stringify comparison unless absolutely necessary... what do you think?

Reply:
I'm trying to figure out when `JSON.stringify(item) === JSON.stringify(items);` would be true. Am I missing something? One is an object. One is a an array.

I'm OK with the change if it's already not working for sure.